include/tvTypes.h

```diff
@@ -336,41 +336,47 @@ typedef enum tvPQModeIndex {
+    PQ_PARAM_ASPECT_RATIO,                      //!< Picture parameter is Aspect ratio
+    PQ_PARAM_COLOR_TEMPERATURE,                 //!< Picture parameter is Colour temperature
+    PQ_PARAM_DIMMINGMODE,                       //!< Picture parameter is Dimming mode
+    PQ_PARAM_LDIM,                               //!< Picture parameter is LDIM. To be deprecated 
+    PQ_PARAM_LOCALDIMMING_LEVEL,                //!< Picture parameter is Local dimming level
+    PQ_PARAM_LOWLATENCY_STATE,                  //!< Picture parameter is Low latency state
+    PQ_PARAM_CMS,                               //!< Picture parameter is CMS. To be deprecated
+    PQ_PARAM_CMS_STATE,                         //!< Picture parameter is CMS. To be deprecated
+    PQ_PARAM_CMS_SATURATION_RED,                //!< Picture parameter is Component saturation red
+    PQ_PARAM_CMS_SATURATION_BLUE,               //!< Picture parameter is Component saturation blue
+    PQ_PARAM_CMS_SATURATION_GREEN,              //!< Picture parameter is Component saturation green
+    PQ_PARAM_CMS_SATURATION_YELLOW,              //!< Picture parameter is Component saturation yellow
+    PQ_PARAM_CMS_SATURATION_CYAN,               //!< Picture parameter is Component saturation cyan
+    PQ_PARAM_CMS_SATURATION_MAGENTA,            //!< Picture parameter is Component saturation magenta
+    PQ_PARAM_CMS_HUE_RED,                       //!< Picture parameter is Component hue red
+    PQ_PARAM_CMS_HUE_BLUE,                      //!< Picture parameter is Component hue blue
+    PQ_PARAM_CMS_HUE_GREEN,                     //!< Picture parameter is Component hue green
+    PQ_PARAM_CMS_HUE_YELLOW,                     //!< Picture parameter is Component hue yellow
+    PQ_PARAM_CMS_HUE_CYAN,                      //!< Picture parameter is Component hue cyan
+    PQ_PARAM_CMS_HUE_MAGENTA,                   //!< Picture parameter is Component hue magenta
+    PQ_PARAM_CMS_LUMA_RED,                      //!< Picture parameter is Component luma red
+    PQ_PARAM_CMS_LUMA_BLUE,                     //!< Picture parameter is Component luma blue
+    PQ_PARAM_CMS_LUMA_GREEN,                    //!< Picture parameter is Component luma green
+    PQ_PARAM_CMS_LUMA_YELLOW,                    //!< Picture parameter is Component luma yellow
+    PQ_PARAM_CMS_LUMA_CYAN,                     //!< Picture parameter is Component luma cyan
+    PQ_PARAM_CMS_LUMA_MAGENTA,                  //!< Picture parameter is Component luma magenta
+    PQ_PARAM_WB_GAIN_RED,                       //!< Picture parameter is Component Red Gain
+    PQ_PARAM_WB_GAIN_GREEN,                     //!< Picture parameter is Component Green Gain
+    PQ_PARAM_WB_GAIN_BLUE,                      //!< Picture parameter is Component Blue Gain
+    PQ_PARAM_WB_OFFSET_RED,                     //!< Picture parameter is Component Red Offset
+    PQ_PARAM_WB_OFFSET_GREEN,                   //!< Picture parameter is Component Blue offset
```

Comment:
Do you have approval from the architects for all these changes? Architect approval is required for all these PR changes.

Reply:
I have sent mail and got approval. Will Send PR and get approval for the same.